posts/2020/12/jupyterlab-winter-theme.md

```diff
@@ -0,0 +1,347 @@
+specific to different types of text and display modes, so you could have a theme that looks like the standard light 
+theme until you enter Presentation mode.
+
+### Color
+
+Less is more. Choosing a color palette of three or fewer hues can be easier to manage and make the whole interface 
+more cohesive since those colors will likely be repeated across the UI. Try it out; it might be surprising how just 
+changing a few color variables can create a very different JupyterLab.
+
+A color's value—or how light or dark it is—also determines contrast. Contrast is key to legibility and creating an 
+experience that includes low-vision users. [WCAG color contrast guidelines](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast) 
+apply to text and graphic interactive 
+elements. There are many tools available, including [downloadable contrast checkers](https://www.paciellogroup.com/color-contrast-checker/) 
+and [web app versions](https://userway.org/contrast/000000/ffffff).
+
+### Text
+
+JupyterLab is full of text, so this can be a place for major changes with very little code. You can easily change font, 
+color, spacing, and size. It's a good idea not have text below 10pt in size, or smaller than the default 
+[`--jp-ui-font-size0`](https://github.com/jupyterlab/jupyterlab/blob/083c65d92686d23b813f0242fca5be3d8b6fae37/packages/theme-light-extension/style/variables.css#L107) 
+(and it's an accessibility recommendation).
+
+### Icons
+
+JupyterLab's icons live in the [packages directory](https://github.com/jupyterlab/jupyterlab/tree/083c65d92686d23b813f0242fca5be3d8b6fae37/packages/theme-light-extension/style/icons) 
+and are part of or based on [Material Icons](https://material.io/resources/icons/?search=clos&icon=warning&style=round). 
+If you want to change or add icons and keep them matching, finding one from this system will fit best. The [Material Design system](https://material.io/design/iconography/system-icons.html) 
+also points out some of their icon design principles which are good to follow if you need to make custom icons that 
+match with the rest. Use SVGs, not PNGs, and remember to give it a [tooltip](https://en.wikipedia.org/wiki/Tooltip). 
+Most of all, make sure to give it an ARIA label (like [this recommendation](https://gomakethings.com/icon-accessibility-and-aria-label/)).
+
+# Modifying the Theme Variables
+
+After each modification the watch process will build the extension for you. 
+No need to stop and restart JupyterLab server; simply refresh the page.
+
+Now we are going to modify some values in the file `variables.css` in our
+project. This file controls many of the colors of JupyterLab, and is a nice place
+to start to change the overall color scheme before doing more detailed
+customisation.
+
+We'll try to update the current theme from orange and blue to more blue-ish tones, 
+which tend to remind me of the holiday
```

Comment:
This section is written in the first person ("me" here, some "I" further down), which is a little odd if the blog post has five authors. Can you change this?

Reply:
done.